packages/rum-core/src/domain/view/viewCollection.ts

```diff
@@ -77,22 +77,26 @@ function processViewUpdate(
       frustration: {
         count: view.eventCounts.frustrationCount,
       },
-      cumulative_layout_shift: view.commonViewMetrics.cumulativeLayoutShift,
-      first_byte: toServerDuration(view.initialViewMetrics.firstByte),
-      dom_complete: toServerDuration(view.initialViewMetrics.domComplete),
-      dom_content_loaded: toServerDuration(view.initialViewMetrics.domContentLoaded),
-      dom_interactive: toServerDuration(view.initialViewMetrics.domInteractive),
+      cumulative_layout_shift: view.commonViewMetrics.cumulativeLayoutShift?.value,
+      cumulative_layout_shift_target_selector: view.commonViewMetrics.cumulativeLayoutShift?.targetSelector,
+      first_byte: toServerDuration(view.initialViewMetrics.navigationTimings?.firstByte),
+      dom_complete: toServerDuration(view.initialViewMetrics.navigationTimings?.domComplete),
+      dom_content_loaded: toServerDuration(view.initialViewMetrics.navigationTimings?.domContentLoaded),
+      dom_interactive: toServerDuration(view.initialViewMetrics.navigationTimings?.domInteractive),
       error: {
         count: view.eventCounts.errorCount,
       },
       first_contentful_paint: toServerDuration(view.initialViewMetrics.firstContentfulPaint),
-      first_input_delay: toServerDuration(view.initialViewMetrics.firstInputDelay),
-      first_input_time: toServerDuration(view.initialViewMetrics.firstInputTime),
-      interaction_to_next_paint: toServerDuration(view.commonViewMetrics.interactionToNextPaint),
+      first_input_delay: toServerDuration(view.initialViewMetrics.firstInputTimings?.firstInputDelay),
+      first_input_time: toServerDuration(view.initialViewMetrics.firstInputTimings?.firstInputTime),
+      first_input_target_selector: view.initialViewMetrics.firstInputTimings?.firstInputTargetSelector,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
      first_input_delay: toServerDuration(view.initialViewMetrics.firstInput?.delay),
      first_input_time: toServerDuration(view.initialViewMetrics.firstInput?.time),
      first_input_target_selector: view.initialViewMetrics.firstInput?.targetSelector,
```